.github/workflows/size-limit.yml

```diff
@@ -3,6 +3,7 @@ on:
   pull_request:
     branches:
       - main
+      - next
```

Comment:
Turns out when we changed branching model we never updated this, so size-limit hasn't been running on most PRs (just the release PRs into main).

I was tempted to just delete the size-limit stuff, as I'm not sure it's really providing that much value.

But figured what they hey, for now I'll just configure it to run on PRs into next.